control/control.go

```diff
@@ -125,21 +127,27 @@ type managesSigning interface {
 	ValidateSignature([]string, string, []byte) error
 }
 
-type ControlOpt func(*pluginControl)
+// PluginControlOpt is the type of pluginControl struture.
+type PluginControlOpt func(*pluginControl)
```

Comment:
Golint does like the name of the variable as the variable is used in the other package as control.ControlOpt, It's stutter to Golint. If you have a better naming option, please share.